sbin/common/common.sh

```diff
@@ -173,7 +173,25 @@ function setBootJdk() {
     fi
 
     echo "Guessing JDK_BOOT_DIR: ${BUILD_CONFIG[JDK_BOOT_DIR]}"
-    echo "If this is incorrect explicitly configure JDK_BOOT_DIR"
+    echo "If this is incorrect explicitly configure JDK_BOOT_DIR using --jdk-boot-dir"
+
+    # Calculate version number of boot jdk
+    export BOOT_JDK_VERSION_NUMBER=$(${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java -XshowSettings:properties -version 2>&1 | awk '/java.specification.version/{gsub(/1\./,"",$3);print $3}')
```

Comment:
I think this will fix it. Unclear why the `awk` (5.1.0) on there isn't accepting `gsub(` properly given that it's in the man page for `awk` in the cygwin enviornment but this less efficient version  seems to work in my cygwin env
```suggestion
    export BOOT_JDK_VERSION_NUMBER=$(${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java -XshowSettings:properties -version 2>&1 | awk '/java.specification.version/{print $3}') | awk -F. '{print$NF}'
```

Reply:
Done